doc/styles/main.css

```diff
@@ -29,49 +29,83 @@ hr {
   border-bottom: 1px solid #ddd;
 }
 
```

Comment:
Can you move the changes to the style to another PR and hopefully show some screenshots of what has changed? Or host on surge.sh the PR result